build.properties

```diff
@@ -1,4 +1,28 @@
+# General Settings
+JAVA_VERSION=9.0.1
+JAVA_VENDOR=oracle
+DISTRIBUTION_NAME=centos
+DISTRIBUTION_RELEASE=7
 DOCKER_IMAGE_REPOSITORY=alfresco-base-java
-DOCKER_IMAGE_TAG=8u161-oracle-centos-7
```

Comment:
This may be overcomplicating things.  Wouldn't we need this same tag building 'logic' duplicated in Bamboo to tag properly in Docker registries?

Reply:
Yes, this tag building logic is duplicated in bamboo. I'll revert to the previous simpler version.

Reply:
It looks like you'll still need the segments to pull from Maven, but it's probably simpler to just duplicate those segments in DOCKER_IMAGE_TAG.

That's more for human's to mess up of course, perhaps we should add a note to the README.

Reply:
Removed